### PR TITLE
Update instructions for Windows [skip ci].

### LINKED ISSUE #1719: Update Windows build instructions.
The README is broken for Windows.

### PR DESCRIPTION
**Pull request by coryan**
Updated the instructions for Windows because the script we recommended no longer exists.

I tested this on a Windows GCE VM, to it "Should Work":tm:

This fixes #1719.

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-cpp/1821)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -263,12 +273,86 @@ You will find compiled binaries in `build-output/` respective to their source pa
...
+    gtest:x64-windows-static ^
+    crc32c:x64-windows-static
+.\vcpkg.exe integrate install
+```
+
+Now clone `google-cloud-cpp`:
+
+```console
+cd ..
+git clone https://github.com/GoogleCloudPlatform/google-cloud-cpp
```
Comment (devbww) [inline README.md]: Does the ".git" extension mean anything in this case?  Even if not, should we include it anyway (like the GitHub "Clone with HTTPS" button does)?

File: README.md [hunk 2]
```diff
@@ -218,11 +218,21 @@ brew install curl cmake libressl c-ares
 
 #### Windows (using vcpkg)
 
+Probably the easiest way to download and compile `google-cloud-cpp` on Windows
+is to use [vcpkg][vcpkg-link], a package manager from Microsoft that downloads
+the source code for a library (and all its dependencies), compiles it, and adds
+the right configuration for DevStudio and/or CMake.
+
+After downloading and compiling `vcpkg`, simply run:
```
Comment (frankyn) [inline README.md]: I may have not said this before, but refrain from using words such as "easiest" and "simple". Comfortable developers may find this path straight forward, but new developers might hit headaches.

File: README.md [hunk 3]
```diff
@@ -263,12 +273,86 @@ You will find compiled binaries in `build-output/` respective to their source pa
...
+Use CMake to create the build files:
+
+```console
+cmake -H. -Bbuild-output -GNinja ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DCMAKE_TOOLCHAIN_FILE="%SOURCE%\vcpkg\scripts\buildsystems\vcpkg.cmake" ^
+    -DVCPKG_TARGET_TRIPLET=x64-windows-static ^
+    -DCMAKE_C_COMPILER=cl.exe ^
+    -DCMAKE_CXX_COMPILER=cl.exe ^
+    -DGOOGLE_CLOUD_CPP_DEPENDENCY_PROVIDER=package
```
Comment (devbww) [inline README.md]: Are those "^" things line continuations?  If so, it looks like we should have one here too.

File: README.md [hunk 4]
```diff
@@ -218,11 +218,21 @@ brew install curl cmake libressl c-ares
...
 ```bash
-set PROVIDER=vcpkg
-set GENERATOR="Visual Studio 15 2017 Win64"
-powershell -exec bypass .\ci\install-windows.ps1
+.\vcpkg.exe install google-cloud-cpp:x64-windows-static
 ```
+
+and then follow the instructions from `vcpkg`. We try to keep the version of
+`google-cloud-cpp` up-to-date, our practice is to submit a PR to update the
+version in `vcpkg` after each release of `google-cloud-cpp`.
```
Comment (frankyn) [inline README.md]: Is the information on updating the version necessary for a Windows developer?  I don't know much about vcpkg but it seems out of scope.

Comment (coryan) [reply README.md]: We had a customer hesitating to use `vcpkg` because they thought it might be too old.  I want to give folks a sense that this is kept reasonably up to date.

File: README.md (hunk 2 above)
Comment (coryan) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (coryan) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (coryan) [reply README.md]: Done.

**Comment by coryan** [comment]
PTAL.

File: README.md (hunk 4 above)
Comment (frankyn) [reply README.md]: Sg, thanks for clarifying.

File: README.md [hunk 5]
```diff
@@ -218,11 +218,21 @@ brew install curl cmake libressl c-ares
 
 #### Windows (using vcpkg)
 
+Probably the easiest way to download and compile `google-cloud-cpp` on Windows
```
Comment (frankyn) [inline README.md]: "Probably the easiesst" -> "A preferred way to download.." Maybe?

**Comment by coryan** [comment]
@frankyn PTAL.
